pyTigerGraph/pyTigerGraph.py

```diff
@@ -960,54 +987,123 @@ def runInterpretedQuery(self, queryText, params=None):
             print(queryText)
         return self._post(self.gsUrl + "/gsqlserver/interpreted_query", data=queryText, params=params, authMode="pwd")
 
-    # Pandas dataframe support =================================================
+    def parseQueryOutput(self, output, graphOnly=True):
```

Comment:
Not clear from the docstring how the output is different from the input. Could you give an example?

Reply:
More details/explanations provided

Reply:
Thanks!